gobblin-core/src/main/java/gobblin/publisher/TimePartitionedDataPublisher.java

```diff
@@ -43,30 +36,4 @@
   public TimePartitionedDataPublisher(State state) throws IOException {
     super(state);
   }
-
-  /**
-   * This method needs to be overridden for TimePartitionedDataPublisher, since the output folder structure
-   * contains timestamp, we have to move the files recursively.
-   *
-   * For example, move {writerOutput}/2015/04/08/15/output.avro to {publisherOutput}/2015/04/08/15/output.avro
-   */
-  @Override
-  protected void addWriterOutputToExistingDir(Path writerOutput, Path publisherOutput, WorkUnitState workUnitState,
```

Comment:
Ok, but is there any loss in functionality my removing this method? Will the `TimePartitionedDataPublisher` still work?